libs/ledger-live-common/src/families/stellar/api/horizon.ts

```diff
@@ -2,37 +2,48 @@ import { LedgerAPI4xx, LedgerAPI5xx, NetworkDown } from "@ledgerhq/errors";
 import { requestInterceptor, responseInterceptor } from "@ledgerhq/live-network/network";
 import type { Account, Operation } from "@ledgerhq/types-live";
 import { BigNumber } from "bignumber.js";
-import StellarSdk, {
+import {
   // @ts-expect-error stellar-sdk ts definition missing?
   AccountRecord,
   NetworkError,
   NotFoundError,
+  Server,
+  HorizonAxiosClient,
+  BASE_FEE,
+  Asset,
+  Operation as StellarOperation,
+  Account as StellarAccount,
+  Transaction as StellarTransaction,
```

Comment:
Not super important, just to avoid confusion with the general pattern that's used in all families (and `StellarOperation` that's defined in `types.ts`)
```suggestion
  Operation as StellarSdkOperation,
  Account as StellarSdkAccount,
  Transaction as StellarSdkTransaction,
```